src/shiny/docker-compose.yaml

```diff
@@ -0,0 +1,32 @@
+version: '2.4'
+services:
+  app:
+    container_name: "application-server"
+    build:
+      context: .
+      dockerfile: Dockerfile
+      args:
+        - BASE_IMAGE=rocker/shiny  
+        # version tag of the rocker/shiny image.
+        - VARIANT=4
```

Comment:
A few questions on this. Why not update the Dockerfile and have `BASE_IMAGE` include the tag:

```
BASE_IMAGE=rocker/shiny:latest
```
If there's a compelling reason not to, then change this from `VARIANT` to `IMAGE_TAG` and the comment is not needed.